config/vufind/config.ini

```diff
@@ -185,6 +185,11 @@ allowSavedSearches = true
 nonJavascriptSupportEnabled = false
 ; Generator value to display in an HTML header <meta> tag:
 generator = "VuFind 10.1"
+; The following setting can be set to true in case a need arises for extended HTML
+; attribute escaping. Default is false. Extended escaping is slower and results in
+; larger HTML responses than normal escaping, so this setting should normally be
+; left disabled.
```

Comment:
From a bit of research, it looks to me like the main reason for escapeHtmlAttr is to provide additional safety in a situation where an HTML attribute is not properly enclosed in quotes. So maybe we should make note here that it's especially important to ensure properly quoted attributes in local templates when leaving this setting false.

Reply:
Right, of course. I revised the comment.